src/helpers/validators/signupValidator.js

```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi';
```

Comment:
I have a file where I implement a logic like this one also. I think we can mix it with this one for us to have an easily understandable codebase

Reply:
```helpers/validation.schemas.js```

Reply:
I don't think it is an issue now since your PR is not merged yet, we will work on that while rebasing.

Reply:
cool